api/views.py

```diff
@@ -1,35 +1,37 @@
+from django.db.models.query import QuerySet
 from rest_framework import authentication, mixins, permissions, viewsets
+from rest_framework.generics import CreateAPIView, RetrieveUpdateAPIView
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
 from accounts.models import Log, User
-from accounts.serializers import LogSerializer
-from todolist import serialzers
+from accounts.serializers import (LogSerializer, UserProfileSerializer,
+                                  UserSerializer)
 from todolist.models import TodoListItem
 from todolist.serialzers import TodoListItemSerializer
 
 
 class LogViewSet(viewsets.ModelViewSet):
-    permission_classes = [permissions.IsAuthenticated]
-    authentication_classes = [authentication.TokenAuthentication]
+    permission_classes = (permissions.IsAuthenticated, )
+    authentication_classes = (authentication.TokenAuthentication, )
     queryset = Log.objects.all()
     serializer_class = LogSerializer
 
 
 class ListUsers(APIView):
-    """
+    '''
     View to list all users in the system.
 
     * Requires token authentication.
     * Only admin users are able to access this view.
-    """
-    permission_classes = [permissions.IsAdminUser]
-    authentication_classes = [authentication.TokenAuthentication]
+    '''
+    permission_classes = (permissions.IsAdminUser, )
+    authentication_classes = (authentication.TokenAuthentication, )
 
     def get(self, request, format=None):
-        """
+        '''
         Return a list of all users.
-        """
+        '''
         usernames = [user.username for user in User.objects.all()]
```

Comment:
can we use `<query>.values('username')`?

also, I would like to ask why we are only returning `usernames`?